run-tests-with-build-kogito-apps-parent/kogito-apps-project-sources-test/pom.xml

```diff
@@ -107,8 +107,6 @@
                 <pomInclude>**/pom.xml</pomInclude>
               </pomIncludes>
               <pomExcludes>
```

Comment:
you can even remove this `pomExcludes` node, wdyt?

Reply:
I wanted to keep it there just to be more explicit, also suggests a place to exclude things in the future.